site/content/en/docs/Installation/_index.md

```diff
@@ -47,7 +47,8 @@ The following table lists recent Agones versions and their corresponding require
 
 | Agones version | Kubernetes version(s) |
 | -------------- | ------------------    |
-| 1.35           | {{% k8s-version %}}   |
+| 1.36           | {{% k8s-version %}}   |
+| 1.35           | 1.26, 1.27, 1.28      |
```

Comment:
The kubernetes versions that 1.35 supports are 1.25 1.26 and 1.27

Reply:
updated!